index.html

```diff
@@ -38,7 +38,133 @@ <h2>LET'S CONNECT</h2>
             </section>
         </main>
         </div>
-    </div>
-    
+        
+    </div>  
+</div>
+
+<section class="hero">
+<div class="card">
+    <img src="img/Snapshoot Portfolio.png" alt="">
+    <h1>Tonic</h1>
```

Comment:
The font size of this div is not according to the suggested in Figma Design. Kindly take a look at it.

| Your Design   | FIGMA Design  |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/61361037/221020132-0ffdb5f4-bccb-4a2a-9382-6e5d6b6ce91b.png)| ![image](https://user-images.githubusercontent.com/61361037/221020079-379ac41e-dc4d-4965-a89d-6a5532b1bef8.png)|